specification/dataprotection/resource-manager/Microsoft.DataProtection/preview/2022-10-01-preview/dataprotection.json

```diff
@@ -3356,6 +3446,267 @@
+          "$ref": "./examples/ResourceGuardProxyCRUD/PutResourceGuardProxy.json"
+        }
+      }
+    },
+    "delete": {
+      "tags": [
+        "DppResourceGuardProxies"
+      ],
+      "operationId": "DppResourceGuardProxy_Delete",
+      "produces": [
+        "application/json"
+      ],
+      "parameters": [
+        {
+          "$ref": "#/parameters/SubscriptionId"
+        },
+        {
+          "$ref": "#/parameters/ResourceGroupName"
+        },
+        {
+          "$ref": "#/parameters/VaultName"
+        },
+        {
+          "name": "resourceGuardProxyName",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        },
+        {
+          "$ref": "#/parameters/ApiVersion"
+        }
+      ],
+      "responses": {
+        "200": {
+          "description": "OK"
+        },
+        "204": {
+          "description": "NoContent"
+        },
+        "default": {
+          "description": "Error response describing why the operation failed.",
+          "schema": {
+            "$ref": "#/definitions/CloudError"
+          }
+        }
+      },
+      "x-ms-examples": {
+        "Delete ResourceGuardProxy": {
+          "$ref": "./examples/ResourceGuardProxyCRUD/DeleteResourceGuardProxy.json"
+        }
+      }
+    }
+  },
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataProtection/backupVaults/{vaultName}/backupResourceGuardProxies/{resourceGuardProxyName}/unlockDelete": {
+    "post": {
+      "tags": [
+        "DppResourceGuardProxies"
+      ],
+      "operationId": "DppResourceGuardProxy_UnlockDelete",
+      "produces": [
+        "application/json"
+      ],
+      "parameters": [
+        {
+          "$ref": "#/parameters/SubscriptionId"
+        },
+        {
+          "$ref": "#/parameters/ResourceGroupName"
+        },
+        {
+          "$ref": "#/parameters/VaultName"
+        },
+        {
+          "name": "resourceGuardProxyName",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        },
+        {
+          "$ref": "#/parameters/ApiVersion"
+        },
+        {
+          "name": "parameters",
+          "in": "body",
+          "description": "Request body for operation",
+          "required": true,
+          "schema": {
+            "$ref": "#/definitions/UnlockDeleteRequest"
+          }
+        }
+      ],
+      "responses": {
+        "200": {
+          "description": "OK",
+          "schema": {
+            "$ref": "#/definitions/UnlockDeleteResponse"
+          }
+        },
+        "default": {
+          "description": "Error response describing why the operation failed.",
+          "schema": {
+            "$ref": "#/definitions/CloudError"
+          }
+        }
+      },
+      "x-ms-examples": {
+        "UnlockDelete ResourceGuardProxy": {
+          "$ref": "./examples/ResourceGuardProxyCRUD/UnlockDeleteResourceGuardProxy.json"
```

Comment:
>examples [](http://example.com/codeflow?start=21&length=8)

add example files wherever new ones are referenced

Reply:
added. these additions are from previous preview version in order to maintain completeness. We aren't making any new changes as part of this PR. Please check the previous preview version here - https://github.com/Azure/azure-rest-api-specs/tree/main/specification/dataprotection/resource-manager/Microsoft.DataProtection/preview/2022-09-01-preview